### ISSUE TITLE
Permission clone combo boxes not keyboard navigable

### ISSUE THREAD

**Issue by ndarilek** [hasura/graphql-engine#2425]
As a screen reader user attempting to use the Hasura web interface, I can't seem to clone permissions. Clicking the Clone button pops up several combo boxes for selection that I cannot tab to. Additionally, none of the tricks I've attempted open the combo boxes for keyboard selection.

Thanks.

**Comment by Dhruvi16** [comment]
I would like to take this up.

**Comment by ringfungi** [comment]
I haven't been able to reproduce this issue, I can tab into the combo boxes (using Google Chrome
Version 85.0.4183.121, Windows 10 x64). Can you give us more details?

**Comment by ndarilek** [comment]
Probably not, sorry. We stopped using Hasura a long time ago in part 
because we needed accessibility, and I'd have to set up an environment 
again.


Might no longer be relevant. I'd also test with a screen reader like 
NVDA, Orca or Voiceover, since this issue was opened in the context of 
multiple blind/low-vision users unable to use the product.

**Comment by NoriSte** [comment]
> I haven't been able to reproduce this issue, I can tab into the combo boxes (using Google Chrome Version 85.0.4183.121, Windows 10 x64). Can you give us more details?

Closing this issue because of this @ringfungi 's comment, feel free to re-open it in the future
